crates/pathfinder/src/storage/schema.rs

```diff
@@ -0,0 +1,2 @@
+pub(crate) mod revision_0001;
```

Comment:
heh I was wondering if you would go with `xx` or `xxx`. But you've gone one order more :D

Reply:
Yeah four... It seems enough :) Took me a while to think why did I instictively put 0001, and I remembered that `git format-patch` does the same.